code/src/components/app-dropdown/AppDropdown.test.tsx

```diff
@@ -0,0 +1,55 @@
+import { screen, fireEvent } from "@testing-library/react";
+import AppDropdown from "@/components/app-dropdown/AppDropdown";
+import { renderWithProviders } from "@/utils/test-utils";
+
+const mockAppDropdownOnSelect = jest.fn();
+
+const options = [
+  { value: "option1", label: "Label 1" },
+  { value: "option2", label: "Label 2" }
+];
+
+describe("AppDropdown block", () => {
```

Comment:
Do we need to call it block here? You can just remove it

Reply:
Word `Block` is redundant in my opinion

Reply:
> Word `Block` is redundant in my opinion

Is it so important `AppDropdown block` or `"AppDropdown` naming for you ?
It's defined with `block` word because it's block of tests

Reply:
It is obvious that `describe` defines a block of tests. In your case in is butter oil

Reply:
> It is obvious that `describe` defines a block of tests. In your case in is butter oil

Thank you! Removed